lua/starfall/libs_cl/render.lua

```diff
@@ -2104,6 +2104,23 @@ function render_library.draw3DQuad(vert1, vert2, vert3, vert4)
 	render_DrawQuad(vert1, vert2, vert3, vert4, currentcolor)
 end
 
+--- Draws a quad.
+-- @param Vector pos Origin of the quad.
+-- @param Vector normal The face direction of the quad.
+-- @param number width The width of the quad.
+-- @param number height The height of the quad.
+-- @param Color? clr The color of the quad.
+-- @param number? rot The rotation of the quad counter-clockwise in degrees around the normal axis. In other words, the quad will always face the same way but this will rotate its corners.
+function render_library.draw3DQuadEasy(pos, norm, width, height, clr, rot)
+	if not renderdata.isRendering then SF.Throw("Not in rendering hook.", 2) end
+
+	pos = vunwrap(pos)
+	norm = vunwrap(norm)
```

Comment:
There's a new pos_vector and norm_vector variable you can use instead which is faster than unwrap.

Reply:
just replace the unwraps with those respectively?

Reply:
No, look at https://github.com/thegrb93/StarfallEx/blob/7406377cbc9abe5c7cd618b5613e928a2e546050/lua/starfall/libs_cl/render.lua#L2358

Reply:
This quickly copies pos into startpos_vec, and you can pass startpos_vec into the glua function.

Reply:
Alright, did it like that